docs/src/examples/sinusoid_example.md

```diff
@@ -0,0 +1,415 @@
+unconstrained_inputs = CES.Utilities.get_inputs(input_output_pairs)
+inputs = Emulators.transform_unconstrained_to_constrained(prior, unconstrained_inputs)
+outputs = CES.Utilities.get_outputs(input_output_pairs)
+```
+
+### Gaussian process
+We will set up a basic Gaussian process (GP) emulator using the [`ScikitLearn.jl`](https://scikitlearnjl.readthedocs.io/en/latest/models/#scikitlearn-models) package or [`GaussianProcesses.jl`](https://stor-i.github.io/GaussianProcesses.jl/latest/). 
+See the [Gaussian process page](https://clima.github.io/CalibrateEmulateSample.jl/dev/GaussianProcessEmulator/) for more information and options, including choice of package and kernels.
+```
+gppackage = Emulators.GPJL()
+gauss_proc = Emulators.GaussianProcess(gppackage, noise_learn = false)
+
+# Build emulator with data
+emulator_gp = Emulator(gauss_proc, input_output_pairs, normalize_inputs = true,  obs_noise_cov = Γ)
+optimize_hyperparameters!(emulator_gp)
+```
+For this simple example, we already know the observational noise `Γ=0.2*I`, so we set `noise_learn = false`. 
+However, for more complicated problems we may want to learn the noise as an additional hyperparameter.
+
+We will check performance of the GP by testing on unseen data. We do this by generating some
+additional data from our true model for a set of 10 parameters.
+```
+amplitude_points = range(0.4, 4, length = 10)
+shift_points = range(-1, 8, length = 10)
+x_test = hcat(shuffle(amplitude_points), shuffle(shift_points))'
+
+# benchmark against true model
+y_true = zeros(2, length(amplitude_points))
+for i in 1:length(amplitude_points)
+    y_true[:, i] = G(x_test[:, i])
+end
+# emulator is built in unconstrained space, we need to first transform our variables
+x_test_unconstrained = Emulators.transform_constrained_to_unconstrained(prior, x_test)
+y_gp, y_gp_covs = Emulators.predict(emulator_gp, x_test_unconstrained, transform_to_real=true);
+
+# This predicts an output and a covariance matrix.
+# Estimate root mean squared error
+rmse_gp = sqrt.(mean((y_gp - y_true).^2, dims=2))
+println("RMSE GP:", rmse_gp)     # [1.0188362036675438; 0.10148451670974754;;]
+```
+
+One metric for assessing emulator performance is to calculate the root mean squared error 
+of our test set, in this case (1.02, 0.10). These errors are relatively small compared against 
+the outputs, around 10%. You could also consider other metrics, such as R^2, mean absolute error or 
+log likelihood. It is also helpful to check emulator performance by plotting the true output 
+against the predicted output. 
+
+![GP_emulator](../assets/sinusoid_GP_emulator.png)
+
+For a perfect GP emulator, our predictions should line up with the true output, along the y=x dashed line.
+We find that this is mostly the case, except for large values of y_1. This is not too surprising, because
+this region is more extrapolation rather than interpolation, as EKI focuses our training data towards the truth
```

Comment:
lets' not use words like extrapolation, and just say "points are further from the training set data"